### ISSUE TITLE
Add alarm without specifying a week day

### ISSUE THREAD

**Issue by eandersons** [SimpleMobileTools/Simple-Clock#213]
It would be nice if a new alarm could be added without specifiying a week day to use it only once when the specified time occurs.

**Comment by tibbi** [comment]
duplicate of #127

**Comment by eandersons** [comment]
Sorry about duplicate!

I would like to point out that this issue is duplicate of #38 and seems that of #58 also.
This issue is only partly duplicate of #127 as #127 is about one time alarm for specific **date** and time which would be a nice feature as well.
